examples/2-orgs-policy-any-no-ca-same-cc/values/orderer.yaml

```diff
@@ -35,7 +35,7 @@ enrollments:
   creds:
   - { name: admin, secret: adminpwd, options: "--id.attrs admin=true:ecert"}
   - { name: user, secret: pwd, options: "--id.type orderer"}
-  csrHost: network-orderer-hlf-ord.orderer
+  csrHost: "network-orderer-hlf-ord.orderer,network-orderer-hlf-ord.orderer.svc.cluster.local"
```

Comment:
![image](https://user-images.githubusercontent.com/660091/110298888-5e418780-7ff5-11eb-8cc7-a84f724a9d98.png)

Je serais pour garder qu'un seul des deux, network-orderer-hlf-ord.orderer.svc.cluster.local dans ce cas pour régler le soucis remonté sur https://github.com/SubstraFoundation/substra-backend/pull/385.

Reply:
It means that we need to update all host values from examples too ! Can you confirm it ?

Reply:
I mean, yes. It seems weird to have a something called A at one place and B at the other pointing to the same service. `sed -i "s/network-orderer-hlf-ord.orderer/network-orderer-hlf-ord.orderer.svc.cluster.local/g"` is the way